content/docs/user-guide/glossary/dvc-metafiles.md

```diff
@@ -0,0 +1,32 @@
+---
+name: 'DVC Metafiles'
+match: ['DVC files', 'files', 'directories']
+tooltip: 'DVC metafiles tooltip...'
+---
+
+# DVC Metafiles
```

Comment:
Yeah now that I see this I think it's best to keep this here and only leave DVC internals in https://dvc.org/doc/user-guide/dvc-files-and-directories which has gotten too long. But I need @shcheklein to confirm on this plan please, since that guide is probably the most important one we have so far.

p.s. if we keep it we can also mention/reiterate on the (versioning through) codification principle here, which will be explained in the new Versioning use case (#1747) — so no need for that concept stand-alone.

Reply:
> I think it's best to keep this here and only leave DVC internals

Agree on that. Will skip over metafiles for now while we wait for confirmation. In latest deploys you can see how it's looking with cache information extracted already.

> if we keep it we can also mention/reiterate on the (versioning through) codification principle here ... no need for that concept stand-alone

Makes sense. A single source keeps things simple.

Reply:
> I think it's best to keep this here and only leave DVC internals in https://dvc.org/doc/user-guide/dvc-files-and-directories which has gotten too long

Ping @shcheklein WDYT?

Reply:
Hi! Update on this one: I did chat with Ivan and we (sort of) agreed to go this way, but will probably change the name to just **DVC Files** here (it may become a section with several pages in the future even), and **DVC Internals** for the guide. No need to do that in this PR; this is just meant as a confirmation.

Reply:
And another update on this: actually it's uncertain whether moving the DVC Files specs from the current guide to the concepts section will be an improvement. And since we want concept pages to be brief, after all this is probably not a good idea. If it has already been done I'll take care of figuring it out, otherwise let's not do this now. Thanks